provider/block_devices_plug.py

```diff
@@ -0,0 +1,436 @@
+"""
+Module for providing interfaces about plugging block devices.
+
+Available classes:
+- BlockDevicesPlug: The class provides interfaces about hot plugging and
+                    unplugging block devices.
+
+Available methods:
+- hotplug_devs_serial: Hot plug the block devices by serial.
+- unplug_devs_serial: Unplug the block devices by serial.
+- hotplug_devs_threads: Hot plug the block devices by threads.
```

Comment:
better to name it `hotplug_devs_threaded`

Reply:
Thanks, have updated.